### DISCUSSION TITLE
[SOLVED] Getting error ValidationException

### DISCUSSION THREAD

**Discussion by sunpale** [codeigniter4/shield#381] [Q&A] [answered]
I have a question, why when I try to save data and perform input validation, I get an error message from shield CodeIgniter\Shield\Exceptions\ValidationException. Even though the form is not related to the shield.
for information I use session-based authentication and apply session filters on all pages

![image](https://user-images.githubusercontent.com/29444319/184617077-79168b1b-e456-46a5-9735-8115aeaf1d11.png)

**Comment by datamweb** [comment]
related to: [forum ](https://forum.codeigniter.com/showthread.php?tid=82568&pid=399187#pid399187)

**Reply by sunpale** [reply]
@datamweb  okay thanks for the advice, for now it can help eliminate the error.

**Reply by datamweb** [reply]
@sunpale 
Can you help us reproduce the error?
What steps have you taken?
Please the minimum code to reproduce?

**Reply by sunpale** [reply]
@datamweb i did nothing to fix the error due to timing issues, i followed what was on the forum by taking a shortcut by setting recordActiveDate to false

**Comment by kenjis** [comment, marked as answer]
I got the cause.

`redirect()->withInput()` or `withErrors()` saves validation errors in the Session.
And the check of the validation errors in Model (`checkQueryReturn()`),
`$this->validation->getErrors()` returns the errors in the Session.

**Reply by kenjis** [reply]
I sent a PR #383
